src/main/java/seedu/address/logic/commands/RenameTagCommand.java

```diff
@@ -0,0 +1,105 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Renames a predefined tag.
+ */
+public class RenameTagCommand extends Command {
+    public static final String COMMAND_WORD = "renametag";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Renames an existing tag.\n"
+            + "Example: " + COMMAND_WORD + " t/Bride's Friend t/Friend";
+
+    public static final String MESSAGE_SUCCESS = "Tag has been renamed.";
+    public static final String MESSAGE_NONEXISTENT_OR_DUPLICATE = "The tag you wish to rename does not exist, "
+            + "or the tag you wish to rename it to already exists.\n";
```

Comment:
Not sure if it is more user friendly to split up the two messages.

Reply:
Will be dealing with this in my next PR as it requires some change of logic that I'm not sure about yet